response_builder/v1/models/base.py

```diff
@@ -131,7 +131,7 @@ class Ballot(BaseModel):
     ballot_paper_id: str = Field()
     ballot_title: str = Field()
     poll_open_date: datetime.date = Field()
-    elected_role: str = Field()
+    elected_role: str = Field(default=None)
```

Comment:
I think this needs to be `Optional[str]` to prevent type errors?

Reply:
:+1: ced00c68

Slightly more generally, for projects like this where we are using types really extensively it would be really useful to look at using something like mypy or pylance in CI to do proper type checks

not today though